### PR TITLE
Add support for Python 3.7 and 3.8, drop EOL 3.4

### PR DESCRIPTION
**Pull request by hugovk**
Also test on Python 3.8 beta, due out in October.

* https://www.python.org/dev/peps/pep-0569/

### CODE REVIEW

**Comment by hugovk** [comment]
Updated, Python 3.8 is now officially out!

File: README.rst [hunk 1]
```diff
@@ -71,7 +71,7 @@ Copyright Jonathan Hartley 2013. BSD 3-Clause license; see LICENSE file.
 Dependencies
 ============
 
-None, other than Python. Tested on Python 2.7, 3.4, 3.5 and 3.6.
+None, other than Python. Tested on Python 2.7 and 3.5+.
```
Comment (wiggin15) [inline README.rst]: I think I prefer to keep the explicit list of tested versions. When future versions get released, we can't say that they are tested until they are.

Comment (hugovk) [reply README.rst]: Updated in https://github.com/tartley/colorama/pull/228/commits/9f03cb6c237ac420e60858f20ee775224cb5e186.

**Comment by wiggin15** [comment]
Thanks!

**Comment by hugovk** [comment]
You're welcome!

**Pull request merged**